tests/foreman/ui_airgun/test_settings.py

```diff
@@ -351,104 +218,15 @@ def test_negative_update_max_trend_param(session, set_original_property_value):
-    """Updates parameter "max_trend" under General tab
-
-    :id: a171d2fc-0e09-4953-a038-42d78171b465
-
-    :expectedresults: Parameter is updated successfully
-
-    :CaseImportance: Critical
-    """
-    property_name = 'max_trend'
-    set_original_property_value(property_name)
-    with session:
-        for param_value in valid_maxtrend_timeout_values():
-            session.settings.update(
-                'name = {}'.format(property_name),
-                param_value
-            )
-            result = session.settings.read('name = {}'.format(property_name))
-            assert result['table'][0]['Value'] == param_value
-
-
-@tier1
-def test_negative_update_idle_timeout_param(session, set_original_property_value):
-    """Updates parameter "idle_timeout" under General tab with
-    invalid values
-
-    :id: 3b09ea93-753e-40d0-a425-8e4548b8181c
-
-    :expectedresults: Parameter is not updated
-
-    :CaseImportance: Critical
-    """
-    property_name = 'idle_timeout'
-    set_original_property_value(property_name)
-    with session:
-        for param_value in invalid_settings_values():
-            with raises(AssertionError) as context:
-                session.settings.update(
-                    'name = {}'.format(property_name),
-                    param_value
-                )
-            assert is_valid_error_message(str(context.value))
-
-
-@tier1
-def test_positive_update_idle_timeout_param(session, set_original_property_value):
-    """Updates parameter "idle_timeout" under Auth tab
-
-    :id: d3188607-0c75-4808-81fe-cc6fea0637fe
-
-    :expectedresults: Parameter is updated successfully
-
-    :CaseImportance: Critical
-    """
-    property_name = 'idle_timeout'
-    set_original_property_value(property_name)
-    with session:
-        for param_value in valid_maxtrend_timeout_values():
-            session.settings.update(
-                'name = {}'.format(property_name),
-                param_value
-            )
-            result = session.settings.read('name = {}'.format(property_name))
-            assert result['table'][0]['Value'] == param_value
-
-
-@tier1
-def test_positive_update_foreman_url_param(session, set_original_property_value):
-    """Updates parameter "foreman_url" under General tab
-
-    :id: c1dd73e0-25f6-4fad-8c8f-b4a39ce108f1
-
-    :expectedresults: Parameter is updated successfully
-
-    :CaseImportance: Critical
-    """
-    property_name = 'foreman_url'
-    set_original_property_value(property_name)
-    with session:
-        for param_value in valid_urls():
-            session.settings.update(
-                'name = {}'.format(property_name),
-                param_value
-            )
-            result = session.settings.read('name = {}'.format(property_name))
-            assert result['table'][0]['Value'] == param_value
-
-
-@tier1
-def test_negative_update_foreman_url_param(session, set_original_property_value):
+@tier2
```

Comment:
this more looks a tier1 test.

Reply:
@oshtaier @vijay8451 As this a negative test and tier1 test. As per your earlier comments, I will be removing this test as well. Please let me know if you have questions

Reply:
same as above